lib/libc/stdio/printf.3

```diff
@@ -105,12 +110,13 @@ and
 .Fn vsnprintf
 write to the character string
 .Fa str ;
-and
 .Fn asprintf
 and
 .Fn vasprintf
 dynamically allocate a new string with
-.Xr malloc 3 .
+.Xr malloc 3
```

Comment:
IMO the previous one, "dynamically allocate a new string with malloc(3)." seems better than the explicit mentioning of "that is stored in ret." (ret is the return pointer, but confusing here).

Reply:
i see that and it does read better having the ret removed, updates will have this change.